Dockerfile

```diff
@@ -0,0 +1,39 @@
+# syntax=docker/dockerfile:1.3
+
+ARG PHP_VERSION=7.4
+
+FROM php:${PHP_VERSION}-cli AS base
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV COMPOSER_FLAGS="--prefer-dist --no-interaction"
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+COPY docker/php/php.ini /usr/local/etc/php/php.ini
+
+RUN apt-get update && apt-get install -y \
+        git \
+        unzip \
+        zlib1g-dev \
```

Comment:
na co je potreba ty `-dev` balicky? zadny PHP extension se tu nebuildi

Reply:
nevim no, je to kopirovane z starej dockerfile